charts/tekton-pipelines/values.yaml

```diff
@@ -66,7 +66,11 @@ buildpacks:
 
             [ -f package.json ] && {
               echo "install node.js dependencies"
-              npm ci --silent
+
+              # set cache
+              npm config set cache $(workspaces.source.path)/.npm --global
+              npm ci
```

Comment:
```suggestion
              npm ci --quiet
```

Reply:
- Frontend team wanted to see the output of the command
- what's interesting with node:16, `--silent` resulted in the build to crash, and without it - worked very well